README.md

```diff
@@ -674,6 +675,22 @@ Its easy to forget full syntax, rather let's create `.vscode/tasks.json` file wi
             },
             "problemMatcher": []
         },
+        {
+            "type": "shell",
+            "label": "Flash project",
+            "command": "STM32_Programmer_CLI",
+            "args": [
+                "--connect",
+                "port=swd",
+                "--download",
+                "build/project-name.elf", // Put here the path to your elf file
```

Comment:
Can we use here cmake plugin to provide name for us? See `launch.json` file how it is done there -> if can work here, we can get rid of template

Reply:
Sure we can, I wasn't aware of that feature! `tasks.json` will become:
```json
        {
            "type": "shell",
            "label": "Flash project",
            "command": "STM32_Programmer_CLI",
            "args": [
                "--connect",
                "port=swd",
                "--download",
                "${command:cmake.launchTargetPath}",
                "-hardRst"
            ],
            "options": {
                "cwd": "${workspaceFolder}"
            },
            "problemMatcher": []
        },
```

I'll rename the template file, revert the changes on `convert.py` and update the readme. Wdyt?

Reply:
Absolutely, I agree. Did you test if it works properly?